core/src/main/java/io/questdb/cairo/wal/seq/SeqTxnTracker.java

```diff
@@ -60,6 +90,30 @@ public long getWriterTxn() {
         return writerTxn;
     }
 
+    public void hadEnoughMemory(CharSequence tableName) {
+        maxRecordedInflightPartitions = 1;
+        walBackoffUntil = -1;
+        if (regulationValue == Integer.MAX_VALUE) {
+            // already at max parallelism, can't go more optimistic
+            return;
+        }
+        if (regulationValue < 0) {
+            // was in backoff, go back to parallelism = 1
+            regulationValue = 1;
+            LOG.info().$("Memory pressure easing off, removing backoff").I$();
```

Comment:
Logging is a bit useless here, table name not specified.
Can we remove logging logic from here and move it to the caller and just return and int value from here to indicate what happened? That would also be testable then

Reply:
We added table name to the call, just missed applying it here.